snapcraft/internal/lxd.py

```diff
@@ -147,3 +134,47 @@ def _wait_for_network(self):
                 if retry_count == 0:
                     raise e
         logger.info('Network connection established')
+
+
+def _get_default_remote():
+    """Query and return the default lxd remote.
+
+    Use the lxc command to query for the default lxd remote. In most
+    cases this will return the local remote.
+
+    :returns: default lxd remote.
+    :rtype: string.
+    :raises snapcraft.internal.errors.SnapcraftEnvironmentError:
+        raised if the lxc call fails.
+    """
+    try:
+        default_remote = check_output(['lxc', 'remote', 'get-default'])
+    except CalledProcessError:
+        raise SnapcraftEnvironmentError(
+             'The lxd package is not installed, in order to use '
+             '`cleanbuild` you must install lxd onto your system. '
+             'Refer to the "Ubuntu Desktop and Ubuntu Server" section on '
+             'https://linuxcontainers.org/lxd/getting-started-cli/'
+             '#ubuntu-desktop-and-ubuntu-server to enable a proper setup.')
```

Comment:
And what if they're using something other than Ubuntu? I think we should instead suggest they install the lxd snap.